config/collectdata/python/write_organ_tags.py

```diff
@@ -48,17 +63,16 @@
 yaml = ruamel.yaml.YAML()
 yaml.indent(sequence=4, offset=2)
 
-with open('neo4j2owl-config.yaml') as file:
+with open(fileName) as file:
```

Comment:
Can you change fileName to file_name?
https://peps.python.org/pep-0008/#function-and-variable-names